sdk/python/tests/components/test_python_op.py

```diff
@@ -106,6 +102,7 @@ def add_two_numbers_indented(a: float, b: float) -> float:
         op = comp.func_to_container_op(func,output_component_file='add_two_numbers_indented.component.yaml')
```

Comment:
Just remove the `,output_component_file='add_two_numbers_indented.component.yaml'` part. It's not needed in this test.

Reply:
SG.

Reply:
Done.